example-kotlin-android/src/main/kotlin/com/vimeo/android/networking/example/kotlin/AccountPreferenceManager.kt

```diff
@@ -0,0 +1,74 @@
+package com.vimeo.android.networking.example.kotlin
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.preference.PreferenceManager
+import com.vimeo.networking.model.User
+import com.vimeo.networking.model.VimeoAccount
+import com.vimeo.networking.utils.VimeoNetworkUtil
+
+/**
+ * Class for persisting the the auth access token (and possibly the user)
+ *
+ * Created by anthonyr on 5/8/17.
+ */
+object AccountPreferenceManager {
+
+    private var sSharedPreferences: SharedPreferences? = null
```

Comment:
Since we use `!!` when accessing this field, I'd assume we'd want it to be annotated `NonNull`. Any use of `!!` would imply non-nullness in my mind.

But also with the `?`, that's saying it's nullable - but you're accessing it like it's not? Not sure what the best thing to do here is.